src/gui/qgsexpressionbuilderwidget.cpp

```diff
@@ -60,11 +62,11 @@ QgsExpressionBuilderWidget::QgsExpressionBuilderWidget( QWidget *parent )
   mValueGroupBox->hide();
 //  highlighter = new QgsExpressionHighlighter( txtExpressionString->document() );
 
-  mModel = new QStandardItemModel();
-  mProxyModel = new QgsExpressionItemSearchProxy();
+  mModel = qgis::make_unique<QStandardItemModel>();
```

Comment:
Can do. I don't think it makes a difference though apparently it was working like this since the beginning of times and the earth as we know it without problems.

Reply:
Yeah. Feel free to leave :P